content/docs/reference/neon-cli.md

```diff
@@ -0,0 +1,224 @@
+---
+title: Neon CLI
+subtitle: Use the Neon CLI to manage Neon projects from the command line
+enableTableOfContents: true
+isDraft: true
+---
+
+Neon offers several methods for working with your projects. Utilizing the Neon Command Line Interface (CLI), you can operate Neon directly from a terminal or via automation. The Neon CLI facilitates numerous functions, such as Neon authentication, project creation and management, role assignment, and more.
+
+## Synopsis
+
+The `neonctl` command can be called from the command line. Without any arguments, it displays command usage and help:
+
+```bash
+neonctl --help
+usage: neonctl <cmd> [args]
+
+Commands:
```

Comment:
Are you sure we want to publish this? It will be changing all the time

Reply:
removed.